cmd/egress/cmd.go

```diff
@@ -158,22 +93,101 @@ are set correctly before execution.
 				config.CaCert = bytes.NewBuffer(cert).String()
 			}
 
-			p := proxy.ProxyConfig{
+			proxy := proxy.ProxyConfig{
```

Comment:
This is a bit confusing as it will overload `proxy` as a var and a package... I think Go can resolve these correctly, however for readability, it might be helpful to rename this var.